pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "chti"
+version = "0.1.0"
+description = ""
+authors = ["Sébastien Eustace <sebastien@eustace.io>"]
```

Comment:
Oh - do we definitely need to change this @lincmba?  Would be embarrassing to share :-)

Reply:
Missed that. Fixing it